src/openforms/registrations/api/urls.py

```diff
@@ -9,6 +14,11 @@
         AllAttributesListView.as_view(),
         name="registrations-attribute-list",
     ),
+    path(
+        "catalogi",
+        CatalogiListView.as_view(),
+        name="catalogi-list",
+    ),
```

Comment:
can you move this down to the plugin urlpatterns, and also move the `informatieobjecttypen` entry?

These are endpoints that are tightly coupled to particular registration plugins and they fit the TODO that's present already.

Reply:
These two endpoints are used by both the ZGW and Objects API registration plugins, that's why I put them here.